modules/auth.js

```diff
@@ -21,95 +20,89 @@ function authenticate (email, password) {
   })
 }
 
-function signupApi (email, password, password_confirmation, newsletter) { // eslint-disable-line
+function signupApi (email, password, passwordConfirmation, newsletter) {
   return apiRequest.post('account/register', {
     account: {
       email,
       password,
-      password_confirmation,
+      passwordConfirmation,
```

Comment:
should be changed to:

`password_confirmation: passwordConfirmation,`

the variable name is important

Reply:
not sure I got it right. you mean the variable name ?
I might have missed something as the var name could be anything ?

Reply:
It's currently calling the api with to register an account with:
```
{
account: {
 ...
passwordConfirmation,
...
}
}
```

but the API looks for:

```
{
account: {
 ...
password_confirmation,
...
}
}
```